lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -90,6 +90,7 @@ class S1ApUtil(object):
     datapath = get_datapath()
     SPGW_TABLE = 0
     LOCAL_PORT = "LOCAL"
+    LOCAL_PORT_NON_NAT_IPV6 = 15577
```

Comment:
This is for mtr port. we should be using port number of 'patch-port' here.

Reply:
What is the significance of `mtr port` and `patch-port`. @pshelar Is there a link where I can read about these two ports? Thanks in advance.

Reply:
mtr port is used for monitoring purpose.
patch port is used for traffic forwarding.